modules/REST/API.php

```diff
@@ -56,7 +56,7 @@ public function init(): void {
 		$this->languages = new V1\Languages( $this->model->languages, $this->model->translatable_objects );
 		$this->languages->register_routes();
 
-		$this->settings = new V1\Settings( $this->model->options );
+		$this->settings = new V1\Settings( $this->model->options, $this->model->languages );
```

Comment:
I would pass only one parameter: `$this->model`.

Reply:
Done in [96edeca](https://github.com/polylang/polylang/pull/1594/commits/96edeca6efffd98bf453b9bb9f1e1de236bdcb31).